src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -283,6 +283,11 @@
             <a i18n
                routerLink="/rgw/multisite">Multisite</a>
           </li>
+          <li routerLinkActive="active"
+              class="tc_submenuitem tc_submenuitem_rgw_buckets">
+            <a i18n
+               routerLink="/rgw/overview">Overview</a>
+          </li>
```

Comment:
Overview should be the first thing in the `Object Gateway` nav list.

Reply:
Done